lib/turkish_support.rb

```diff
@@ -1,9 +1,9 @@
-require "turkish_support/version"
-require "turkish_support/constants"
-require "turkish_support/string_helpers"
-require "turkish_support/string_methods"
-require "turkish_support/array_methods"
-require "turkish_support/destructives"
+require 'turkish_support/version'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.